dist/LICENSE

```diff
@@ -0,0 +1,390 @@
+
+If you have any suggestions, additions, comments, or questions, please
+let [21]me know.
+
+[22]Douglas C. Schmidt
+  _________________________________________________________________
+
+Back to the [23]ACE home page.
+
+References
+
+   1. http://www.cs.wustl.edu/~schmidt/ACE.html
+   2. http://www.cs.wustl.edu/~schmidt/TAO.html
+   3. http://www.dre.vanderbilt.edu/CIAO/
+   4. http://www.dre.vanderbilt.edu/cosmic/
+   5. http://www.dre.vanderbilt.edu/~schmidt/
+   6. http://www.cs.wustl.edu/~schmidt/ACE-members.html
+   7. http://www.wustl.edu/
+   8. http://www.uci.edu/
+   9. http://www.vanderbilt.edu/
+  10. mailto:doc_group@cs.wustl.edu
+  11. http://www.cs.wustl.edu/~schmidt/ACE-users.html
+  12. http://www.cs.wustl.edu/~schmidt/ACE.html
+  13. http://www.cs.wustl.edu/~schmidt/TAO.html
+  14. http://www.dre.vanderbilt.edu/CIAO/
+  15. http://www.dre.vanderbilt.edu/~schmidt/DOC_ROOT/DAnCE/
+  16. http://www.dre.vanderbilt.edu/cosmic/
+  17. http://www.dre.vanderbilt.edu/
+  18. http://www.isis.vanderbilt.edu/
+  19. http://www.cs.wustl.edu/~schmidt/doc-center.html
+  20. http://www.cs.wustl.edu/~schmidt/commercial-support.html
+  21. mailto:d.schmidt@vanderbilt.edu
+  22. http://www.dre.vanderbilt.edu/~schmidt/
+  23. http://www.cs.wustl.edu/ACE.html
+
+
+---------------------------------------------------------------------------
+The MIT License (http://opensource.org/licenses/MIT)
+---------------------------------------------------------------------------
+
+Apache Geode bundles the following files under the MIT License:
+
+  - libxml2 (http://xmlsoft.org),
+    Copyright (C) 1998-2012 Daniel Veillard.  All Rights Reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+---------------------------------------------------------------------------
+ANTLR 2 License (http://www.antlr2.org/license.html)
+---------------------------------------------------------------------------
+
+Apache Geode bundles the following files under the ANTLR 2 License:
```

Comment:
It is "public domain" but does have additional clauses so I included the entire license. Are you saying to just state it is public domain "period".